src/trunk.c

```diff
@@ -7958,13 +7966,16 @@ trunk_print_pivots(platform_log_handle *log_handle,
    for (uint16 pivot_no = 0; pivot_no < trunk_num_pivot_keys(spl, node);
         pivot_no++)
    {
-      key               pivot = trunk_get_pivot(spl, node, pivot_no);
+      key pivot = trunk_get_pivot(spl, node, pivot_no);
+      // Used to format generated string for negative / positive infinity keys.
+      char buf[PIVOT_KEY_OUTBUF_LEN + 1];
+      snprintf(buf, sizeof(buf), "%-24s", key_string(spl->cfg.data_cfg, pivot));
+
       trunk_pivot_data *pdata = trunk_get_pivot_data(spl, node, pivot_no);
       if (pivot_no == trunk_num_pivot_keys(spl, node) - 1) {
          platform_log(log_handle,
-                      "| %.*s | %12s | %12s | %11s | %9s | %5s | %5s |\n",
-                      24,
-                      key_string(spl->cfg.data_cfg, pivot),
+                      "| %s | %12s | %12s | %11s | %9s | %5s | %5s |\n",
```

Comment:
This change, and the introduction of `PIVOT_KEY_OUTBUF_LEN` and `char buf` on L7971 [etc.] is needed because the +/-ve infinity key length is not quite 24 bytes. So the outputs come out mis-aligned (slightly) with the older code. 

The outputs will be correctly aligned with this new code, as seen below:

Generated using: `./build/release/bin/unit/splinter_test test_splinter_print_diags`

```
Page type: trunk, Node addr=393216
{
----------------------------------------------------------------------------------------------------
|          |     addr     |   next addr  | height |   gen   | pvt gen |                            |
|  HEADER  |--------------|--------------|--------|---------|---------|----------------------------|
|          |       393216 |            0 |      1 |       0 |       8 |                            |
|--------------------------------------------------------------------------------------------------|
|                                       PIVOTS                                                     |
|--------------------------------------------------------------------------------------------------|
|         pivot key        |  child addr  |  filter addr | tuple count | kv bytes  |  srq  |  gen  |
|--------------------------|--------------|--------------|-------------|-----------|-------|-------|
| (negative_infinity)      |       524288 |   6200492032 |      149741 |  11841707 |    -1 |     6 |
| 0x20001c5108e65507000000 |       548864 |   6201278464 |      149188 |  11782153 |    -1 |     2 |
| Full key: 20001c51 08ffffffe65507 00000000 00000000 00000000 00000000
| 0x3ffc153e7bc3f0e2000000 |       532480 |   6202064896 |       37150 |   2937469 |    -1 |     7 |
```